app/Console/Components/Npm/FetchTotals.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Console\Components\Npm;
+
+use App\Events\Packagist\TotalsFetched;
+use Developmint\NpmStats\NpmStats;
+use GuzzleHttp\Client;
+use Illuminate\Console\Command;
+
+class FetchTotals extends Command
+{
+    protected $signature = 'dashboard:fetch-npm-totals';
+
+    protected $description = 'Fetch totals for all our NPM packages';
+
+    public function handle()
+    {
+        $npmStats = new NpmStats(new Client());
+
+        $packageList = collect([
```

Comment:
This is boring list. Put it in  a function called `getPackageNames` at the bottom of the file.

Reply:
Done ☺️